### PR TITLE
python3Packages.yalesmartalarmclient: init at 0.3.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python module to interface with Yale Smart Alarm Systems

https://github.com/domwillcode/yale-smart-alarm-client

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 113791` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.yalesmartalarmclient</li>
    <li>python38Packages.yalesmartalarmclient</li>
    <li>python39Packages.yalesmartalarmclient</li>
  </ul>
</details>

File: pkgs/development/python-modules/yalesmartalarmclient/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  src = fetchFromGitHub {
+    owner = "domwillcode";
+    repo = "yale-smart-alarm-client";
+    rev = "v${version}";
+    sha256 = "0fscp9n66h8a8khvjs2rjgm95xsdckpknadnyxqdmhw3hlj0aw6h";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  # Projec has no tests
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/yalesmartalarmclient/default.nix:20]:
```suggestion
  # Project has no tests
```

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113791` at 308d4d10 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/1c9a671c9859c2b9fdfcaee9d506fdad">python37Packages.yalesmartalarmclient</a></li></li>
    <li><a href="https://gist.github.com/f1c8e28abf5426cf023423c9bb81884b">python38Packages.yalesmartalarmclient</a></li></li>
    <li><a href="https://gist.github.com/da1fd8b348850bdb9b1dfb1fef7977f1">python39Packages.yalesmartalarmclient</a></li></li>
  </ul>
</details>

File: pkgs/development/python-modules/yalesmartalarmclient/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/yalesmartalarmclient/default.nix]: Thanks, fixed.

**Comment by fabaff** [comment]
@GrahamcOfBorg build python38Packages.yalesmartalarmclient python39Packages.yalesmartalarmclient

**Pull request merged**